### DISCUSSION TITLE
Link to commit seems to be broken

### DISCUSSION THREAD

**Discussion by phil-hotchkiss** [mia-platform/community#395] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

12.2.0

### Description

When using the Last Deployed Configuration link on the deploy page, the integration with Github appears to be broken. 

![image](https://github.com/mia-platform/community/assets/136851567/aa6a8a69-52e7-4461-bc48-c325bb744867)

This does a link with an href="href="https://api.github.com/repos/ORG/REPO/git/commits/COMMITID"" and opens a new window. Github returns a 404 NOT FOUND when visiting this url. However, if I were to visit https://github.com/ORG/REPO/commit/COMMITID, I am able to load the page no problem. 

I feel like the api.github.com link used to work, but maybe github has changed the way it works for visiting that with a session from the browser or something? I am able to hit the same api.github.com route using postman with a PAT and get a valid response. 

Let me know if you think this is an issue on my side or a new bug in Mia

### Actual Outcome

I receive a 404 from github when attempting to click the link for the Last Deployed Configuration

### Expected Outcome

I expect to be able to view the commit in github when clicking the link for the Last Deployed Configuration

**Comment by fredmaggiowski** [comment]
Mmmh this seems like a bug in the link creation @marcofilippi @jacoposte let's track it in backlog and have a look at it, thanks! 

> I feel like the api.github.com link used to work, but maybe github has changed the way it works for visiting that with a session from the browser or something?

@phil-hotchkiss this might be true, regardless I'd move from using the api domain to using the browser one to be more resilient, thanks for reaching out!

**Comment by mattia-fumo** [comment]
Hi,
this error is probably due to a misconfiguration of the `Base URL` of the provider, which is used to render the link in question. Please make sure it is set to `https://github.com` if it is a github provider.

**Reply by phil-hotchkiss** [reply]
Hi Mattia, the BaseUrl is already set to `https://github.com`

![image](https://github.com/mia-platform/community/assets/136851567/dab9615c-1b34-4c27-a547-59fa6ea193cc)

**Comment by phil-hotchkiss** [comment, marked as answer]
Issue seems fixed in more recent versions of Mia platform
